content/en/docs/guides/dualstack.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Dualstack Backends"
+description: >-
+     Explains the dualstack backend feature in gRPC.
+---
+
+### Overview
+
+In cases where an endpoint (i.e., a gRPC server) has both an IPv4 and
+IPv6 address, it is useful for clients to know both of those addresses,
+so that they can reach the endpoint on either of them.  Because only
+one of the two protocols may actually be routable between a given
+client and endpoint, clients need to try both addresses to see which
+one will work.  To minimize connection attempt latency, it is also
+desirable for clients to attempt to connect to multiple addresses in
```

Comment:
"it is also desirable" doesn't say what is actually done or who is responsible for it. "To minimize connection attempt latency, gRPC clients will attempt..."?